policyengine_us/variables/gov/states/il/dhs/tanf/il_tanf_payment_level.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class il_tanf_payment_level(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = (
+        "Illinois Temporary Assistance for Needy Families (TANF) payment level"
+    )
+    unit = USD
+    definition_period = MONTH
+    reference = "https://www.law.cornell.edu/regulations/illinois/Ill-Admin-Code-tit-89-SS-112.251"
+    defined_for = StateCode.IL
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(period).gov.states.il.dhs.tanf.payment_level
+        fpg = spm_unit("spm_unit_fpg", period)
+
+        # Regular family, 0.3 * fpg
+        # Child only family, 0.3 * 0.75 * fpg, when adult receives SSI
+        # Adult only family, 0.3 * 0.25 * fpg, when child receives SSI
```

Comment:
```suggestion
        # The payment levels can vary by ssi eligiblity:
        # If the parent is receiving SSI, the payment level is 75% 
        # if the child is receiving SSI, the payment level is 25%  
```

Lets add an issue for this - or we can add it in this PR - leaving it up to you